src/util/config/config.cpp

```diff
@@ -254,6 +254,13 @@ namespace dxvk {
     { R"(\\The Convenience Store Yakin Jiken\.exe$)", {{
       { "d3d11.enableRtOutputNanFixup",     "True" },
     }} },
+    /* Art of Rally                               */
+    { R"(\\artofrally_demo\.exe$)", {{
```

Comment:
`{ R"(\\artofrally(_demo)?\.exe$)", {{` should make this one if.

Reply:
Done